libs/wingc/src/lib.rs

```diff
@@ -201,15 +210,26 @@ fn add_builtin(name: &str, typ: Type, scope: &mut Scope, types: &mut Types) {
 		.expect("Failed to add builtin");
 }
 
-pub fn compile(source_file: &str, out_dir: Option<&str>) -> Result<CompilerOutput, Diagnostics> {
+pub fn compile(source_path: &Path, out_dir: Option<&Path>) -> Result<CompilerOutput, Diagnostics> {
+	assert!(source_path.is_file(), "Source path must be a file");
+	assert!(
+		source_path.extension().unwrap() == "w",
+		"Source file must have .w extension",
+	);
+	assert!(source_path.is_absolute(), "Source path must be absolute");
```

Comment:
This is exposing a very easily fixed bug in the playground :)

https://github.com/winglang/wing/blob/main/apps/wing-playground/worker.js#L48

In this PR could you update this to be "/code.w" instead of "code.w"?
Otherwise the assertion here fails.